### PR TITLE
[#82211636] Upgrade embers

### PR DESCRIPTION
**Pull request by tonywok**
This pull request is an attempt to upgrade to ember 1.8.1 and ember data 1.0.0.beta.14. We are desperately in need of this - each day we don't, upgrading is getting harder.

:rabbit: :cake: 

Once this is merged into master, we'll be primed for merging master into the `ember-cli` branch.

### CODE REVIEW

**Comment by mikem** [comment]
I'm not seeing where the two new radio button components are used. Why are they there?

Otherwise, looks OK to me. :+1:

**Comment by neo-tahi** [comment]
":ship: it" — @DTrejo

**Comment by mikem** [comment]
Oh, and the text area component too.

**Comment by rizwanreza** [comment]
I wonder if the radio-button component and text-area component override the default components provided by Ember for these.

I am gonna merge this, as it's working fine well otherwise. 

/cc @tonywok

**Comment by tonywok** [comment]
@mikem @rizwanreza Saw those too. Looks like the `DecisionRadioButton` inherits from the `ETahi.RadioButton`. I thought those looked fishy too.

File: spec/features/journal_administration_spec.rb [hunk 1]
```diff
@@ -67,7 +67,10 @@
         role = journal_page.add_role
         role.name = "whatever"
         role.save
-        expect(role).to have_name("whatever")
+        # NOTE: `expect(role).to have_name("whatever")` fails.
+        # Re-finding it works.
+        new_role = journal_page.find_role("whatever")
```
Comment (tonywok) [inline spec/features/journal_administration_spec.rb]: @rizwanreza Anything come to mind when looking at this? I wasn't very satisfied with having to do this, but felt it was for the greater good.

Comment (mikem) [inline spec/features/journal_administration_spec.rb]: I'm not sure about this issue myself, but I think it's fine to leave a comment and move on.

**Pull request merged**